.github/workflows/compare_translations.py

```diff
@@ -114,6 +134,30 @@ def check_translations():
         sys.exit(0)
 
 
+def main():
+    """
+     Parse command-line arguments, check for the existence of the specified directory,
+    and call check_translations with the provided or default directory.
+
+    """
+    parser = argparse.ArgumentParser(
+        description="Check and print missing translations for all non-default languages."
+    )
+    parser.add_argument(
+        "--directory",
+        type=str,
+        nargs="?",
+        default=os.path.join(os.getcwd(), "lang"),
+        help="Directory containing translation files(relative to the root directory).",
+    )
+    args = parser.parse_args()
+
+    if not os.path.exists(args.directory):
+        print(f"Error: The specified directory '{args.directory}' does not exist.")
+        exit(1)
```

Comment:
Refer to the other uses of exit in your script.

Reply:
Please manually test your code for the potential error conditions

Reply:
sorry for the silly mistake.I have fixed it